### DISCUSSION TITLE
Do I get charged for Google Maps usage if I use OpenStreetMaps?

### DISCUSSION THREAD

**Discussion by bimix** [react-native-maps/react-native-maps#4682] [Q&A]
### Summary

I have managed to get `react-native-maps` up and running. I am able to show Google Maps on it.

Now I just need to know 2 things:

1. Where to get the OpenStreetMap API from (I couldn't find it anywhere on the website even after registering there)?
2. Will I get charged for Google Maps usage if I use OpenStreetMaps in the `react-native-maps` package?

### Reproducible sample code

```TSX
import MapView, { MAP_TYPES, UrlTile } from 'react-native-maps';

export default class Map extends React.Component {
   render() {
      return (
         <View style={styles.container}>
            <MapView
               mapType={MAP_TYPES.STANDARD}
               rotateEnabled={false}
               style={styles.map}
               showsUserLocation>
               <UrlTile                    
                  urlTemplate={"https://www.openstreetmap.org/#map=8/56.584/11.294"}                    
                  maximumZ={19}                  
                  flipY={false}
               />
            </MapView>
         </View>
      );
   }
}
```

### Steps to reproduce

simple RN app with react-native-maps installed and run it.

### Expected result

OpenStreetMap usage shouldn't affect Google Maps usage.

### Actual result

Cant find OpenStreetMap API
Docs dont show if I will be charged by Google Maps if I use OpenStreetMap instead.

### React Native Maps Version

1.4.0

### What platforms are you seeing the problem on?

Android

### React Native Version

0.71.2

### What version of Expo are you using?

Not using Expo

### Device(s)

NLS-NFT10 Android 9

**Comment by RyanTG** [comment]
This seems like it should be a Discussion - https://github.com/react-native-maps/react-native-maps/discussions - and not an Issue labeled “bug”.

1. are you talking about OSM tilehosting? If so, I believe your options are to pay for something like mapbox or maptiler if you want OSM tiles, or host them yourself. You can’t just link to the OSM site to load their tiles.
2. Google maps doesn’t charge for their mobile sdk.
